flutter_local_notifications/lib/src/platform_specifics/android/enums.dart

```diff
@@ -91,8 +94,7 @@ class Importance {
   static const Importance max = Importance(5);
 
   /// All the possible values for the [Importance] enumeration.
-  static List<Importance> get values =>
-      <Importance>[unspecified, none, min, low, defaultImportance, high, max];
+  static List<Importance> get values => <Importance>[unspecified, none, min, low, defaultImportance, high, max];
```

Comment:
It seems that my android studio did not pickup your formatting style. Where do I find the config for the line with?